luaui/Widgets/Include/AtlasOnDemand.lua

```diff
@@ -628,12 +628,16 @@ local function MakeAtlasOnDemand(config)
 	function AtlasOnDemand:RenderTasks()
 		if self.hastasks then 
 			if next(self.renderImageTaskList) then 
-				gl.RenderToTexture(self.textureID, self.RenderImageTasks, self)
+				if gl.RenderToTexture ~= nil then
```

Comment:
this is the only one I'm not sure on; this lets the code still run, but no-op if it can't render. We might want to handle this earlier and never even call this file though.

Reply:
Let it run no-op, and return the default uv coords of 0,0,1,1 as on a failed allocation.

Reply:
I'm not sure if I understand exactly; do you mean have `AtlasOnDemand:AddImage(...)` and `AtlasOnDemand:AddText(...)` check if `gl.RenderToTexture` exists, and return `{x = 0,X = 1,y = 0,Y = 1,w = 1,h = 1, id = id}` if it doesn't?

Reply:
done!